cva6/docs/UVM_verif_env.md

```diff
@@ -0,0 +1,227 @@
+<!-- Output copied to clipboard! -->
```

Comment:
This comment header looks like a copy-and-paste from something else.  Please remove it.  In it's place you should add a license header that looks something like this:

```
   Copyright (c) 2023 10x-Engineers

   SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
```

Reply:
Thanks @ranaadeel-10x.  The license header looks good. The license header itself should be in a comment block.  Sorry I was not clear about that.

Reply:
I have included the license header in comment block in this commit with hash 6018dc8.